Kickstarter-iOS/Features/Dashboard/Controller/DashboardViewController.swift

```diff
@@ -198,6 +215,48 @@ internal final class DashboardViewController: UITableViewController {
     }
   }
 
+  private func updateTableViewBottomContentInset() {
+    if let deprecationWarningHostingController = deprecationWarningHostingController {
+      self.tableView.contentInset
+        .bottom = deprecationWarningHostingController.view.bounds
```

Comment:
Found that the `depecrationWarningHostingController.view.bounds` always returns the `height` and all other dimensions as `.zero` if `viewWillAppear` is called, so need to wait until `viewDidAppear` is called for the views to have correct dimensions. That's why the `self.tabBarController.tabBar.bounds.height` was needed - because it existed before `viewWillAppear` on this view controller.

Reply:
nice